lib/turn.rb

```diff
@@ -0,0 +1,26 @@
+class Turn
+
+  attr_reader :guess,
+              :card
+
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    if @guess == @card.answer
+      true
+    else
+      false
+    end
+  end
```

Comment:
You could slim this up quite a bit by saying 

```
def correct? 
 @ guess == @card.answer
end 
```

We could do this becuase `==` will automatically return true or false for us when it's evaluated.